languages/en.lang

```diff
@@ -67,7 +67,7 @@ $PALANG['pMain_create_alias'] = 'Create a new email forward.';
 $PALANG['pMain_create_mailbox'] = 'Create a new mailbox.';
 $PALANG['pMain_sendmail'] = 'Send an email to one of your newly created mailboxes.';
 $PALANG['pMain_password'] = 'Change the password for your admin account.';
-$PALANG['pMain_viewlog'] = 'View the log files.';
+$PALANG['pMain_viewlog'] = 'View PostfixAdmin logs.';
```

Comment:
Note: You'll need to update all the `*.lang` files, either by updating the texts (if you speak 35 languages ;-) or at least by letting `language-update.sh` add a TODO note.